R/shiny-getters.R

```diff
@@ -69,6 +69,7 @@
 #' @param body_value `character` or `JS_EVAL`, the **body** of a JavaScript
 #'   function that Vega will use to handle the signal or event; this function
 #'   must return a value
+#' @param id `character`, optional module id
```

Comment:
I would call the new argument `moduleId` - I know the [modules documentation](https://shiny.rstudio.com/articles/modules.html) uses `id`, but in the absence of a compelling reason otherwise, I prefer to be more descriptive.

Reply:
Sure, I don't like id either :)